endpointmanager/pkg/capabilityhandler/capabilityreceiver.go

```diff
@@ -84,41 +100,26 @@ func saveMsgInDB(message []byte, args *map[string]interface{}) error {
 }
 
 // CapabilityReceiver receives the capability statement from the queue and adds it to the database
-func CapabilityReceiver(store endpointmanager.FHIREndpointStore) {
-	// Get the config information that can then be used below
-	err := config.SetupConfig()
-	failOnError(err)
-
-	// Set up the queue for sending messages
-	qUser := viper.GetString("quser")
-	qPassword := viper.GetString("qpassword")
-	qHost := viper.GetString("qhost")
-	qPort := viper.GetString("qport")
-	qName := viper.GetString("capquery_qname")
-	messageQueue, channelID, err := queue.ConnectToQueue(qUser, qPassword, qHost, qPort, qName)
-	failOnError(err)
-	log.Info("Successfully connected to Queue!")
-	defer messageQueue.Close()
+func CapabilityReceiver(store endpointmanager.FHIREndpointStore,
```

Comment:
(style comment)

Consider renaming.

`CapabilityReceiver(...)` makes me think I'll be getting an object back that I will then call methods on vs a method itself. I recommend making more verby (`ReceiveCapabilityStatements(...)`)

Reply:
👍